src/envelope.rs

```diff
@@ -48,11 +50,44 @@ pub trait MessageEnvelope: Send {
     ) -> BoxFuture<'a, ()>;
 }
 
+#[cfg(feature = "with-tracing-0_1")]
+#[derive(Clone)]
+struct Instrumentation {
+    parent: Span,
+    _waiting_for_actor: Span,
+}
+
+#[cfg(feature = "with-tracing-0_1")]
+impl Instrumentation {
+    fn new<A: Actor, M>() -> Self {
```

Comment:
I believe the `Actor` bound is unnecessary.

Reply:
The other option is bounding with `'static` I think - it's used by the `type_name` in this function

Reply:
I'd prefer that.

Reply:
Nevermind, `type_name` doesn't need any bound :smile: